README.md

```diff
@@ -134,6 +134,15 @@ POST headers:
 
 POST request body:
 - `domain`: the project domain (this must be the domain associated with the project API key)
+- `publication`: configuration for publication
+  - `protocol`: configurations for specific publication protocols
+    - `http`: configuration specific to the `HTTP` protocol
+      - `enable`: boolean, if protocol will be used, default value is `true`
+      - `dnsupdate`: boolean, if DNS records will be updated and deleted, default value is `true`. Set to `false` if manual DNS `A` and `AAAA` entries are configured for `@` host.
```

Comment:
This is too implementation-specific for a config. Delete.

Reply:
Rename to:
```
`purgeIfDisabled`: boolean, if set to true, existing A and AAAA records will be purged when `enable` is set to `false`. This has no effect when `enable` is set to `true`
```

You may also consider renaming all booleans to past tense, like `enabled`, `disabled`, `isUpdated`, and `isDeleted`.

Reply:
Renamed